Documentation/install/upgrade.rst

```diff
@@ -520,14 +520,19 @@ Removed resource fields
 
 .. _1.7_required_changes:
 
-IMPORTANT: Changes required before upgrading to 1.7.0
+IMPORTANT: Changes required before upgrading to 1.7.x
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. warning::
 
-   Do not upgrade to 1.7.0 before reading the following section and completing
+   Do not upgrade to 1.7.x before reading the following section and completing
    the required steps.
 
+   In particular, if you are using network policy and upgrading from 1.6.x or earlier
+   to 1.7.x or later, you MUST read the ConfigMap section below about the
```

Comment:
Ideally, we could add a link to jump directly to the section? In case it moves for whatever reason in the future, and does not remain “below”.